src/components/TitlesCarousel/TitlesCarousel.js

```diff
@@ -45,8 +49,14 @@ export const MovieCarousel = async ({ movies, id }) => {
     });
 
     return `
+    ${Button({
+        onClick: () => redirect(`${route}`),
```

Comment:
this button doesn't work, there is literally error in the console cuz you didn't imported that function xD

Reply:
everything done